src/security_utilities_rust/src/microsoft_security_utilities_core/identifiable_secrets.rs

```diff
@@ -514,7 +630,7 @@ fn get_csrandom_bytes(key_length_in_bytes: u32) -> Vec<u8>
     let mut rng = ChaCha20Rng::from_entropy();
     let mut random_bytes: Vec<u8> = Vec::new();
 
-    for i in 0..key_length_in_bytes
+    for _i in 0..key_length_in_bytes
```

Comment:
>_i [](http://example.com/codeflow?start=8&length=2)

this resolves an 'unused param' error I assume?

Reply:
Exactly.